contracts/utility/NFTCatalog.cdc

```diff
@@ -9,6 +9,7 @@ import MetadataViews from "./MetadataViews.cdc"
 //
 // To make an addition to the catalog you can propose an NFT and provide its metadata.
 // An Admin can approve a proposal which would add the NFT to the catalog
+
 pub contract NFTCatalog {
```

Comment:
We'll probably need to chat with Amit and Bilal about getting this and the FT catalog up to date too because I assume that is required for this, right?

Reply:
NFTCatalog and NFTCatalogAdmin are in main branch, but aren't dependencies for NFTStorefront v1 or v2. Regardless, yes they'll need to be updated

Reply:
I made an issue for their updates here - https://github.com/onflow/nft-catalog/issues/147

Since they're not dependencies for v1 or v2, I'm hoping the NFTCatalog contract updates won't be a blocker to getting this set of contracts updated.